addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -64,28 +65,24 @@ func (r *AntreaConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, err
 	}
 
+	annotations := antreaConfig.GetAnnotations()
+	if _, ok := annotations[constants.TKGAnnotationTemplateConfig]; ok {
+		log.Info(fmt.Sprintf("resource '%v' is a config template. Skipping reconciling", req.NamespacedName))
```

Comment:
Warning should be more appropriate?

Reply:
? don't think so. this is pretty common. Why  a warning?